_pages/security/pii.md

```diff
@@ -3,25 +3,28 @@ title: Personally Identifiable Information
 parent: Security
 ---
 
-**Sensitive PII** is information which if lost, compromised, or disclosed without authorization, could result in substantial harm, embarrassment, inconvenience, or unfairness to an individual. Sensitive PII **cannot be on a system unless it has received an [Authority to Operate](../../ato).** If you are uncertain if the information you want to work with is Sensitive PII in your context, speak with 18F Infrastructure beforehand. Some notes:
+***...and the Privacy Act of 1974.***
 
-* This process is not analogous to the [PIA](../../laws/pia/)/[SORN](../../laws/sorn/) process – those must be handled separately.
-* Broad safe harbor granted for when users may accidentally put their PII in; it must be actually requested.
+**Personally Identifiable Information (PII)** is any information that can distinguish or trace an individual, either alone or when combined with other information. **Sensitive PII** is information that, if misused, could cause substantial harm, embarrassment, inconvenience, or unfairness to a person. Sensitive PII cannot be on a system unless it has received an [Authority to Operate](../../ato/). If you are uncertain if the information you want to work with is Sensitive PII in your context, speak with [the GSA Privacy Office](https://insite.gsa.gov/portal/category/533866) or 18F Infrastructure beforehand. Some notes:
+
+* This process is not analogous to the [PIA](../../laws/pia/)/[SORN](../../laws/sorn/) process – those must be handled separately and, if necessary, should be initiated as close to the start of your project as possible.
+* Collecting some PII that is ancillary to your system's purpose, for example in a search box, does not necessarily trigger [the Privacy Act](https://www.gsa.gov/portal/content/104250). If however, for example, your system is using contact information to communicate with people, it is processing PII.
```

Comment:
@vzvenyach Richard believes this is the part the Office of General Counsel was concerned about, so hopefully his update is satisfactory?